test/unit/helpers/admin/admin_govspeak_helper_test.rb

```diff
@@ -93,15 +93,14 @@ class Admin::AdminGovspeakHelperTest < ActionView::TestCase
   end
 
   test "should allow attached images to be embedded in admin html" do
-    images = [OpenStruct.new(alt_text: "My Alt", url: "/image.jpg")]
+    images = [OpenStruct.new(alt_text: "My Alt", url: "https://some.cdn.com/image.jpg")]
     html = govspeak_to_admin_html("!!1", images)
-    assert_select_within_html html, ".govspeak figure.image.embedded img[src=?]", Whitehall.public_asset_host + "/image.jpg"
+    assert_select_within_html html, ".govspeak figure.image.embedded img[src=?]", "https://some.cdn.com/image.jpg"
   end
 
   test "prefixes embedded image urls with asset host if present" do
```

Comment:
The name no longer seems to match the behaviour, might be worth renaming this test?

Reply:
Good spot, @adrianclay. I'll update the test name.